pkg/test/masteruserrecord/master_user_record.go

```diff
@@ -91,6 +91,11 @@ func NewMasterUserRecord(t *testing.T, userName string, modifiers ...MurModifier
 			Labels: map[string]string{
 				templateTierHashLabelKey(DefaultNSTemplateTier.Name): hash,
 			},
+			OwnerReferences: []metav1.OwnerReference{
+				{
+					BlockOwnerDeletion: func(b bool) *bool { return &b }(true),
```

Comment:
is it something you could test, to ensure it cannot be removed by accident in the future?